index.yaml

```diff
@@ -642,7 +642,7 @@ entries:
       repository: file://./charts/sachet
     description: The manager for third-party webhook receivers used in Prometheus
       Alertmanager
-    digest: 7d164438ac543b36e1b2f7aed50c7b45551464a357e60ac42a79160f934c3558
+    digest: ad5d4a31db37f433b3381981b8ff6e26d421f5b1920101c81e452e4e2760fc5a
```

Comment:
This is expected since we updated the content of the chart, so a new digest value is generated.